db/schema.rb

```diff
@@ -350,6 +351,23 @@
     t.index ["spent"], name: "index_credits_on_spent"
   end
 
+  create_table "curated_cluster_articles", force: :cascade do |t|
+    t.bigint "article_id"
+    t.datetime "created_at", null: false
+    t.bigint "curated_cluster_id"
+    t.datetime "updated_at", null: false
+    t.index ["article_id"], name: "index_curated_cluster_articles_on_article_id"
+    t.index ["curated_cluster_id"], name: "index_curated_cluster_articles_on_curated_cluster_id"
```

Comment:
interesting that these ```t.index```s were added--I have not seen that before

Reply:
My understanding is that these are referencing the foreign keys.